src/reducers/secrets.js

```diff
@@ -80,9 +80,40 @@ function isFetching(state = false, action) {
 function errorMessage(state = null, action) {
   switch (action.type) {
     case 'SECRETS_FETCH_FAILURE':
-    case 'SECRET_DELETE_FAILURE':
     case 'SECRET_CREATE_FAILURE':
       return action.error;
+    case 'SECRET_DELETE_FAILURE':
+      return action.error;
+    case 'SECRETS_FETCH_REQUEST':
+    case 'SECRETS_FETCH_SUCCESS':
+    case 'CLEAR_SECRET_ERROR_NOTIFICATION':
+      return null;
+    default:
+      return state;
+  }
+}
+
+function createSuccessMessage(state = null, action) {
+  switch (action.type) {
+    case 'SECRET_DELETE_SUCCESS':
+    case 'SECRET_CREATE_SUCCESS':
+      return action.success;
```

Comment:
Once the switch statement reaches the case you can just return true here and probably false instead of null below.

Reply:
Done 😸